README.md

```diff
@@ -0,0 +1,123 @@
+  </p>
+</div>
+
+This module bundle helps developers quickly add blog articles to Apostrophe 3 websites. It provides the blog piece type (`@apostrophecms/blog`) as well as a special page type (`@apostrophecms/blog-page`) for editors to create a blog.
+
+## Installation
+
+To install the module, use the command line to run this command in an Apostrophe project's root directory:
+
+```
+npm install @apostrophecms/blog
+```
+
+## Usage
+
+Configure the blog modules in the `app.js` file:
+
+```javascript
+require('apostrophe')({
+  shortName: 'my-project',
+  modules: {
+    // The main blog piece type module
+    '@apostrophecms/': {},
+    // The blog page module
+    '@apostrophecms/blog-page': {}
+  }
+});
+```
+
+### Enable the page type
+
+To enable the blog page type for editor to select, add it to the `@apostrophecms/page` configuration:
+
+```javascript
+// modules/@apostrophecms/page/index.js
+module.exports = {
+  options: {
+    types: [
+      {
+        name: '@apostrophecms/home-page',
+        label: 'Home'
+      },
+      // Adding the blog page type
+      {
+        name: '@apostrophecms/blog-page',
+        label: 'Blog Page'
+      }
+    ]
+  }
+};
+```
+
+**Note:** The index page template (`index.html`), filters template partial (`filters.html`), and show page template (`show.html`) are primarily meant as a starting point for a project. They demonstrate much of the available template data, but developers will almost always want to override them to implement proper styles and layout.
+
+### Filtering by year, month, and day
+
+The default field `releaseDate` sets the date of the blog article. By default, only past articles are displayed to the user. In the Apostrophe UI, all articles are shown to the editors and admin.
+
+The blog page module, `@apostrophecms/blog-page`, provides query filters to refine blog results by year, month, and day. These are primarily used for index page filters (see the `filters.html` file), but can also be used in REST API requests and server-side queries. blogs that span multiple consecutive days are included in results if the query is at least partially included in their date span.
+
+| Filter Name | Description          | Expected Format |
+| ----------- | -------------------- | --------------- |
+| `year`      | Filter by blog year  | `YYYY`          |
+| `month`     | Filter by blog month | `YYYY-MM`       |
+| `day`       | Filter by blog day   | `YYYY-MM-DD`    |
+
+### Multiple blog piece types
+
+Sometimes a website needs multiple, distinct types of blogs. If the blog types can be managed together, it might be easiest to [add a new field](https://v3.docs.apostrophecms.org/guide/content-schema.html#using-existing-field-groups) and [query builder](https://v3.docs.apostrophecms.org/reference/module-api/module-overview.html#queries-self-query) to customize blog views. But if the blog types should be managed completely separately, it may be better to create separate piece types for each.
```

Comment:
say "blog posts" whenever referring to the individual pieces, "blogs" when referring to the pages.

Reply:
Done